app/models/c2dns.py

```diff
@@ -1,4 +1,4 @@
-from distutils import util
+import distutils
```

Comment:
Not really sure what the "correct" way is, but my local was breaking because of this.